README.md

```diff
@@ -414,6 +414,19 @@ for other options).</p>
 at the end of the command can be used, but that should cover most possible use cases.</p>
     </td>
  </tr>
+ <tr>
+    <td><tt>extraunloadoptions</tt></td>
+    <td>No</td>
+    <td><tt>ESCAPE</tt> <tt>MANIFEST</tt> </td>
```

Comment:
Minor subtlety: I wouldn't necessarily say that these are defaults for this setting because if the user sets `extraunloadoptions` to some other value it doesn't override / remove `ESCAPE` and `MANIFEST`. Note that `extracopyoptions`' doc says "No Default" so I wonder if we should do the same here in order to avoid confusion.